Gemfile

```diff
@@ -16,9 +16,11 @@ gem 'coffee-rails', '~> 4.2'
 gem 'jbuilder', '~> 2.5'
 gem 'bootsnap'
 
+
 group :development, :test do
   gem 'pry'
   gem 'figaro'
+  gem 'faraday'
```

Comment:
This is fine for the practice assessment, just keep in mind that for apps deployed to prod we will need faraday to be outside of the `:development, :test` group.